integration-templates/discourse/nango.yaml

```diff
@@ -0,0 +1,17 @@
+integrations:
+    discourse:
+        syncs:
+            users:
+                description: |
+                    Fetches a list of active users from Discourse.
+                endpoint: /discourse/users
+                sync_type: full
+                runs: every 1 hour
```

Comment:
track_deletes: true

Reply:
Thank you Khaliq. Updated with [commit](https://github.com/NangoHQ/nango/pull/2722/commits/4809e695a54651800f1f90a89dbf86d0880b5335)